### PR TITLE
Add hls options and docs, fixing security vulnerability

### PR DESCRIPTION
**Pull request by maxkahan**
This replaces the PR from my fork of the repo, #205. Adds HLS options, documents previously undocumented methods, updates the insecure jQuery version the sample app was using, and includes some minor refactoring/cleaning up.

### CODE REVIEW

**Comment by maxkahan** [comment]
Worth also noting that I've made all of your suggested changes from #205 @jeffswartz and have removed the breaking change. My current intent is to merge all the new PRs into the dev branch and create a minor version.

**Comment by maxkahan** [comment]
Have also added the minor cleanup change mentioned in #202 to this PR in order to get this approved more quickly.

**Comment by maxkahan** [comment]
Have also added a fix for a security vulnerability (specifiying a version of the rsa dependency >=4.7)

File: opentok/opentok.py [hunk 1]
```diff
@@ -1309,7 +1311,8 @@ def start_broadcast(self, session_id, options, stream_mode=BroadcastStreamModes.
             If you include RTMP streaming, you can specify up to five target RTMP streams. For
             each RTMP stream, specify 'serverUrl' (the RTMP server URL), 'streamName' (the stream
             name, such as the YouTube Live stream name or the Facebook stream key), and
-            (optionally) 'id' (a unique ID for the stream)
+            (optionally) 'id' (a unique ID for the stream). You can optionally specify lowLatency or
+            DVR mode, but these options are mutually exclusive.
```
Comment (jeffswartz) [inline opentok/opentok.py:1315]: It is not clear to me from these docs what the `outputs` object looks like. Maybe we should include a code example:

```            "outputs": {
                "hls": {
                    "lowLatency": True,
                    "dvr": True
                    },
                "rtmp": [
                    {
                        "id": "foo",
                        "serverUrl": "rtmp://myfooserver/myfooapp",
                        "streamName": "myfoostream",
                    },
                    {
                        "id": "bar",
                        "serverUrl": "rtmp://mybarserver/mybarapp",
                        "streamName": "mybarstream",
                    },
                ],
            },
```
For a docs example, see https://github.com/opentok/Opentok-Python-SDK/blob/main/opentok/opentok.py#L1117-L1120.

**Review by jeffswartz — APPROVED** [review]

File: opentok/opentok.py (hunk 1 above)
Comment (maxkahan) [reply opentok/opentok.py]: Have added this now

**Pull request merged**